bazel/execution/parse_test.go

```diff
@@ -0,0 +1,104 @@
+package execution
```

Comment:
fwiw: this is a 'micro test' which would not be necessary if we had component testing at the bzutil/main.go level testing the getOperation(), etc. functions.  (Note: I also am guilty of bypassing the component tests for the batch upload/download functionality)